pkgs/development/interpreters/nextflow/default.nix

```
@@ -0,0 +1,43 @@
+{ lib,
+  stdenv,
+  fetchurl,
+  makeWrapper,
+  jre,
+  wget,
+  which,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nextflow";
+  version = "21.10.6";
+
+  src = fetchurl {
+    url = "https://github.com/nextflow-io/nextflow/releases/download/v${version}/nextflow-${version}-all";
+    sha256 = "0l9hi51vrhvfx3px2pxw7lp4h21n8ks50x4icfk3hbgl2hwf7fvx";
+  };
+
+  nativeBuildInputs = [makeWrapper];
+  buildInputs = [jre wget which];
+
+  dontUnpack = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    chmod +x $src
+    mv $src $out/bin/$pname
```

```suggestion
    install -Dm755 $src $out/bin/nextflow
```
Usage of `pname` here makes it a bit harder to read IMO, with no benefits.